ui/component/or-asset-viewer/src/index.ts

```diff
@@ -29,7 +30,7 @@ import i18next from "i18next";
 import {styleMap} from "lit-html/directives/style-map";
 import {classMap} from "lit-html/directives/class-map";
 
-export type PanelType = "property" | "location" | "attribute" | "history" | "chart";
+export type PanelType = "property" | "location" | "attribute" | "history" | "chart" |  "info" | "group";
```

Comment:
Looks good, you might have noticed that the panel type "info" used to exist but was requested to be removed because it didn't show any info that wasn't shown somewhere else

Reply:
I did, in the AssetViewerConfig.
Don's designs in #140 do show an info panel though, so that's the reason I put it back.

Reply:
The panel title `Info` shown in #140 is actually the "attribute" panel as user notes should be an attribute of the `Group` asset....there could be any number of attributes defined for a given group asset.

Reply:
Alright, perfect, I'll remove it just now.